shell/packages/sandstorm-ui-topbar/topbar.js

```diff
@@ -159,8 +183,53 @@ Template.sandstormTopbar.events({
     Template.instance().data.closePopup();
   },
 
+  "click .toggle-sidebar": function (event) {
+    var topbar = Template.instance().data;
```

Comment:
Maybe, though it's a little trickier than "just add a CSS animation to the show/hide of the sidebar" because you also have to resize the rest of the main-content.  Also, the main-content resizes horizontally, whereas we'd probably want to show the sidebar dropping from the top (or raising up into it), since the control for toggling it lives in the topbar.  I wasn't sure what a coordinated animation should look like, so I punted. :)

Also since this winds up reflowing all the grain frames, animating them is pretty expensive here.

I can try a couple options and see if anything feels better - I'd probably try "just animate sidebar and let content reflow immediately) and "try animating them both at the same time in perpendicular directions."  I'm sensitive to the performance implications when there's a bunch of grains, though.